harness/determined/_data_layer/_data_layer.py

```diff
@@ -175,6 +176,13 @@ def cache_dataset(
         self._decorator_used = True
         dataset_version += "_train" if self._training else "_val"
 
+        # Track usage
+        # Temporary measure to determine whether to deprecate data layer/yogadl dependency
```

Comment:
I'd recommend avoiding placing temporal comments in the code, because they have a habit of becoming out of date.  Temporal things make more sense in the commit message, which is more point-in-time by nature.

Reply:
removed this comment and edited the commit message.